### ISSUE TITLE
Some crm_pdf/cr_text tests failing

### ISSUE THREAD

**Issue by sckott** [ropensci/crminer#46]
Probably related to many recent changes in crm_text and crm_pdf, but haven't been able to sort out whats going on. Seems fine when commenting out the vcr usage though, so maybe something to do with file caching/writing to disk.

**Comment by mark-fangzhou-xie** [comment]
I wonder if the following example falls into this category? My package version is: `[1] crminer_0.3.4.95`.

```
> l <- crm_links("10.1023/a:1009865221699")
> l
$unspecified
<url> http://academic.oup.com/rof/article-pdf/3/3/343/26321080/3-3-343.pdf
> t <- crm_text(l, "pdf", overwrite_unspecified = T, useragent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1 Safari/605.1.15")
Downloading pdf...
> t
[1] "~/Library/Caches/R/crminer/3-3-343.pdf"
```

The return value of `crm_text` becomes the path of cached PDF file, and no longer the full text extracted content.

**Comment by sckott** [comment]
Yeah, i think thats related. Its not surprising this is happening due to complexity of the problem. thanks for the comment

**Comment by mark-fangzhou-xie** [comment]
I understand. There are lots of changes that happened recently and we all certainly hope that this package could be more helpful for everyone potentially.

**Comment by sckott** [comment]
I can no longer replicate the problem you're having above - let me know if you have the same problem after installing the latest version from here

**Comment by mark-fangzhou-xie** [comment]
As mentioned in #49, the problem disappeared once I removed all the cached PDFs and updated the package to the newest development version (though I'm not sure what caused the problem to begin with). 

Anyway, thanks a lot! I closed #49 as well.

**Comment by sckott** [comment]
Great, glad it works now
